packages/documentation/src/stories/components/linkarea/linkarea.stories.ts

```diff
@@ -62,27 +39,16 @@ export default meta;
 function renderLinkarea(args: Args) {
   return html`
     <post-linkarea>
-      <div class="card">
-        <div class="card-body">
-          <h5>Title</h5>
-          <p>This card demonstrates the link area functionality. Click anywhere to navigate to the first link, unless a specific link is marked with data-link.</p>
-          <a class="card-link" href="${args.anchorDefaultLink}">Link text</a>
-          <a
-            class="card-link"
-            href="${args.anchorSpecifiedLink}"
-            data-link=${args.dataLink ? '' : nothing}
-            >More links</a
-          >
-        </div>
+      <div class="palette palette-alternate p-32 rounded-8">
```

Comment:
Can you set the class attribute directly on the `post-linkarea`? Then the `post-linkarea` needs a `display: block`.